checks/branch_protected_test.go

```diff
@@ -15,7 +15,10 @@
 package checks
```

Comment:
you can also add e2e tests in the e2e/ folder. You can launch `make e2e` with an GITHUB_AUTH_TOKEN set for scorecard repo. But you should be able to locally test only the branch protection e2e test (I forgot the exact go command :-))

Reply:
want to do this in this PR or in a follow-up one? Are there blockers for this? (I know sometimes it's hard to find a repo that actually passes all the tests)